AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/consts/physics.py

```diff
@@ -0,0 +1,28 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Holds constants used across both hydra and non-hydra scripts.
+"""
+
+"""
+PhysX Components
```

Comment:
Discoverability and usability.

Decentralization of these consts makes it difficult to find if a specific const exists. In fact, I've seen duplication of consts in several places. Additionally, if you do find a specific const elsewhere in the tests, it may not be properly set up to easily use them and can result in some very ugly import pathing and and setting up the lib package, which could potentially change how things are accessed by tests already using them (yes, I've encountered this...).